interfaces/repo_test.go

```diff
@@ -582,6 +673,21 @@ func (s *RepositorySuite) TestAddSlotStoresCorrectData(c *C) {
 	c.Assert(slot, DeepEquals, s.slot)
 }
 
+func (s *RepositorySuite) TestAddSlotParallelInstance(c *C) {
+	c.Assert(s.testRepo.AllSlots(""), HasLen, 0)
+	err := s.testRepo.AddSlot(s.slot)
+	c.Assert(err, IsNil)
```

Comment:
Would love to have a "c.Assert(s.testRepo.AllSlots(""), HasLen, 1)" here too (same as above)